content/blog/platform-decay.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Platform decay, decentralized marketplaces, and Urbit"
+date = 2020-05-08
+description = "Urbit is calm computing. Calm commerce follows naturally. "
+[extra]
+author = "Nicholas Simmons"
+ship = "~simfur-ritwed"
+image ="https://media.urbit.org/site/posts/essays/platform-decay-preview.png"
++++
+
+<br>
+
+<img class="ba" src="https://media.urbit.org/site/posts/essays/platform-decay.png">
+
+Platform decay, decentralized marketplaces, and Urbit
+
+The current model followed by almost all social media services is one of easy and free participation in exchange for advertising and data collection. If you’re a fan of decentralized projects like Urbit, or just someone who pays attention to the ongoing debate over social media incentives, you’re likely to be familiar by now with a central critique of this model. There are historical reasons for why we ended up here: ad-serving and user-tracking technology outpaced subscription and micro-payment models because of recalcitrance by banks and regulatory hassles that prevented ‘real money’ from easily integrating into websites. Instead, we pay in eyeballs and analytical data. 
+
+When we take a moment to think about possible alternatives, we usually fixate on that original bad bargain—’free’ in exchange for ads—-before lamenting that first-mover advantage, critical masses of users, and sophisticated algorithms have made it very hard to see successful challenges to the reign of sites like Facebook and Twitter. We’d be happy to pay a few pennies to have an ad-free, data-secure alternative. But this critique misses something larger. There are, in fact, plenty of colossal Internet businesses which actually do charge people money to participate: e-commerce platforms. Some of these enable the sale of virtual goods and content (Steam, Patreon, Spotify), some physical goods (Ebay, Etsy), and some both (Amazon, Kickstarter). Because these commercial platforms had to figure out how to make online payments easy and secure for mass audiences for their core business to work, they avoided the total dependency on ad revenue that social media sites have always had. Unfortunately, several factors have conspired to push e-commerce platforms closer and closer to a social media-style business model—by manipulating user behavior and prioritizing ad revenue and branding deals over providing a fair marketplace for ordinary buyers and sellers. 
```

Comment:
check dashes in first sentence of 19. I can't tell if they're em dashes? otherwise i think it looks good. i've read it so many times my eyes are kinda crossing on this one.